.changeset/few-geese-retire.md

```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
+"storybook-app": patch
```

Comment:
With this PR you are adding a new component, I think it's a new non breaking feature and so a `minor` change, not a `patch`

```suggestion
"nextjs-website": minor
"storybook-app": minor
```